### ISSUE TITLE
Error with Anaconda Navigator Launcher

### ISSUE THREAD

**Issue by olzam** [ContinuumIO/anaconda-issues#1931]
This is what I get:

Navigator Error

An unexpected error occurred on Navigator start-up

Report

Please report this issue in the anaconda issue tracker

Main Error

string indices must be integers, not str
Traceback

Traceback (most recent call last):
  File "C:\Users\Marcos\Anaconda2\lib\site-packages\anaconda_navigator\exceptions.py", line 75, in exception_handler
    return_value = func(*args, **kwargs)
  File "C:\Users\Marcos\Anaconda2\lib\site-packages\anaconda_navigator\app\start.py", line 115, in start_app
    window = run_app(splash)
  File "C:\Users\Marcos\Anaconda2\lib\site-packages\anaconda_navigator\app\start.py", line 58, in run_app
    window = MainWindow(splash=splash)
  File "C:\Users\Marcos\Anaconda2\lib\site-packages\anaconda_navigator\widgets\main_window.py", line 160, in __init__
    self.api = AnacondaAPI()
  File "C:\Users\Marcos\Anaconda2\lib\site-packages\anaconda_navigator\api\anaconda_api.py", line 1205, in AnacondaAPI
    ANACONDA_API = _AnacondaAPI()
  File "C:\Users\Marcos\Anaconda2\lib\site-packages\anaconda_navigator\api\anaconda_api.py", line 65, in __init__
    self._conda_api = CondaAPI()
  File "C:\Users\Marcos\Anaconda2\lib\site-packages\anaconda_navigator\api\conda_api.py", line 1622, in CondaAPI
    CONDA_API = _CondaAPI()
  File "C:\Users\Marcos\Anaconda2\lib\site-packages\anaconda_navigator\api\conda_api.py", line 340, in __init__
    self.set_conda_prefix()
  File "C:\Users\Marcos\Anaconda2\lib\site-packages\anaconda_navigator\api\conda_api.py", line 489, in set_conda_prefix
    self.ROOT_PREFIX = info['root_prefix']
TypeError: string indices must be integers, not str

**Comment by goanpeca** [comment]
@olzam it seems that conda is not properly set on your path. If you open a terminal, and type

```
$ source activate root
$ anaconda-navigator
```

What happens?

**Comment by olzam** [comment]
Thank you, Gonzalo. I've just reinstalled Anaconda and it runs fine. Greetings!

**Comment by ayushsaxena1997** [comment]
Hello goanpeca,
I had exactly same problem, I did what you said and it worked. However when I restart Anaconda (after closing the previous session as well as the terminal that invoked it) I get the same problem again.
This is what it looked like when I followed you.

`ayush@ayush-Inspiron-3442:~$ source activate root
(root) ayush@ayush-Inspiron-3442:~$ anaconda-navigator
2017-06-10 16:35:30,176 - ERROR download_api._download:192
Invalid url https://continuum.io/sites/default/files/Minne.png

Segmentation fault (core dumped)
(root) ayush@ayush-Inspiron-3442:~$`

Will I have to do this every time I want to use Anaconda? Or there is a permanent fix for this which I am unaware of? 
Thanks a lot!
PS: I am using Ubuntu 14 (new-to-linux user) and this problem popped up after I updated-upgraded through my terminal (or maybe I think so).
